csfieldguide/chapters/content/en/computer-graphics/sections/graphics-transformations.md

```diff
@@ -599,7 +599,7 @@ The original interactives in this section stopped working with the release of ve
 Replacements are in development and should be fully released soon:
```

Comment:
I think this panel could probably be removed now?

Reply:
Hmm, maybe, there's still 1138